notebooks/code_sharing/llm/initial_rag_support.ipynb

```diff
@@ -0,0 +1,957 @@
+       "      <td>[0.014880189672112465, 0.03505474328994751, 0....</td>\n",
+       "      <td>Our governance framework for managing AI risks...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                      id  \\\n",
+       "0   881f050a-874d-4863-a5ce-f802cf4469a4   \n",
+       "10  7ae0aeca-4fe5-495b-b95a-2a24a8f828e0   \n",
+       "13  1e1b0a2a-1d01-4a7b-af63-53a397c90263   \n",
+       "16  4023a94a-41c6-4390-9adc-babd738aafcb   \n",
+       "\n",
+       "                                         RFP_Question  \\\n",
+       "0   Please share your experience with developing A...   \n",
+       "10  What measures do you employ to ensure your LLM...   \n",
+       "13  Describe your strategy for integrating LLMs in...   \n",
+       "16  Can you discuss your governance framework for ...   \n",
+       "\n",
+       "                                           RFP_Answer  \\\n",
+       "0   Our company has 15 years of experience in deve...   \n",
+       "10  We prioritize transparency and explainability ...   \n",
+       "13  Our approach involves conducting a thorough an...   \n",
+       "16  We have established an AI Risk Council that pl...   \n",
+       "\n",
+       "                           embedding_model_prediction  \\\n",
+       "0   [0.006856707856059074, -0.04714655876159668, 0...   \n",
+       "10  [0.010077687911689281, 0.02444615587592125, 0....   \n",
+       "13  [0.010061484761536121, 0.022379260510206223, 0...   \n",
+       "16  [0.014880189672112465, 0.03505474328994751, 0....   \n",
+       "\n",
+       "                                 rag_model_prediction  \n",
+       "0   We have a strong track record in developing AI...  \n",
+       "10  We prioritize transparency by incorporating ex...  \n",
+       "13  Our strategy for integrating Large Language Mo...  \n",
+       "16  Our governance framework for managing AI risks...  "
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vm_test_ds.df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Experiment with some RAGAS Metrics\n",
+    "\n",
+    "Below I am just experimenting to see how the RAGAS metrics can work with the `RAGModel` instance. This is not a full implementation of the RAGAS metrics but just a poc. We'll want to make this work in a more general way so that the columns can be properly mapped from the user-provided `predict_col` or the default `predict_col` to the column names that RAGAS expects i.e. `question`, `contexts`, `answer`, `ground_truth`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'result_df' is not defined",
```

Comment:
result_df is not defined.

Reply:
clearing the outputs for now... we can get the ragas metrics working properly in the next pr :)